### DISCUSSION TITLE
kustomize v4 with --enable-helm?

### DISCUSSION THREAD

**Discussion by joebowbeer** [argoproj/argo-cd#6538] [Q&A] [answered]
With kustomize v4 coming in argocd 2.1, I wonder if there is any recommendation regarding the use of kustomize to render helm charts?

kustomize v4 provides an [updated helm generator](https://newreleases.io/project/github/kubernetes-sigs/kustomize/release/kustomize/v4.1.0), enabled by the `--enable-helm` option.

https://github.com/argoproj/argo-cd/pull/6059

**Comment by wtam2018** [comment, marked as answer]
It looks setting `kustomize.buildOptions` https://argoproj.github.io/argo-cd/user-guide/kustomize/#kustomize-build-optionsparameters in `argocd-cm` ConfigMap will work.

**Reply by iamraj007** [reply]
This saved me some research, thanks @wtam2018 .
